Makefile

```diff
@@ -180,10 +180,12 @@ setup: .env
 	@mkdir -p ./mysql/data
 	@mkdir -p ./redis
 
+ACCELERATE_VERSION:=$(shell git describe --tags --abbrev=0)
 build: shutdown-vms delete-vms setup
+	@echo $(ACCELERATE_VERSION)
```

Comment:
I'd like to see this removed. It feels like a debugging leftover from AC-7217.

Reply:
Either remove, or add some text such as "Using accelerate version: " so it's obvious what we're seeing. (It's not totally insane to confirm the version that you're building!)